randomstringutils.go

```diff
@@ -101,7 +102,21 @@ Returns:
 	error - an error stemming from an invalid parameter within underlying function, RandomSeed(...)
 */
 func RandomAlphaNumeric(count int) (string, error) {
-	return Random(count, 0, 0, true, true)
+	RandomString, err := Random(count, 0, 0, true, true)
+	if err != nil {
+		return "", fmt.Errorf("Error: %s", err)
+	}
+	match, _ := regexp.MatchString("([0-9]+)", RandomString)
```

Comment:
Yeah, a panic-on-error is probably a good approach here. But I don't know that that would necessarily prevent us from merging.

@dbarranco would you mind capturing the error here and just doing something like:

```go
if err != nil {
  panic(err)
}
```

That way if another dev (ahem, most likely me) comes along and breaks the regexp, we'll get an early warning.